src/vs/base/parts/quickinput/browser/quickInputList.ts

```diff
@@ -362,6 +362,14 @@ export class QuickInputList {
 		return Event.map(this.list.onDidChangeSelection, e => ({ items: e.elements.map(e => e.item), event: e.browserEvent }));
 	}
 
+	get scrollTop() {
+		return this.list.scrollTop;
```

Comment:
Unsure about the list-widget but for the tree there is `getViewState` which captured scroll position and selection. I happily use that for the outline when switching between editors - might be an option for the list too...

Reply:
I didn't see a `getViewState` equivalent in my look around the list-widget but good to know!